### PR TITLE
Rewrite district official registration

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.  Note that I merged in #115.

File: src/regdistrictofficial.js [hunk 1]
```diff
@@ -1,34 +1,93 @@
...
+            choices[i] = new Choice(district.id, district.name);
+        }
 
-            next: function(choice) {
-                if(choice.value === 'next') {
-                    return 'state_rdo_exit';
-                } else {
-                    return 'state_rdo_exit';
+        return new PaginatedChoiceState(name, {
+            question: "Please enter your district name.",
```
Comment (imsickofmaps) [inline src/regdistrictofficial.js:20]: Is "enter" what it currently is, even though its a choice? If so, I might ask Lauren if we can change...

**Comment by imsickofmaps** [comment]
change the utils placing as discussed then :+1: if travis happy

File: src/regdistrictofficial.js (hunk 1 above)
Comment (gsvr) [inline src/regdistrictofficial.js:20]: Yes, that's the text being used in production now.  I agree, "select" / "choose" would be better.

**Comment by gsvr** [comment]
Will implement utils placing in #117 since there's quite a bit of refactoring going on with the changing of objects to functions for go.rdo, go.rht, etc.

**Pull request merged**